readyplayerme/rpm-unreal-next-gen-sdk
# [PHO-342] Add functionality to remove asset in samples

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

Previously, there was no way to remove an asset after you select it. As a user what you would expect is, if you click an asset like a Top, it would show as selected in the UI and then load the asset. If you click the same one again the UI button will update to indicate it is deselected however the asset is still equipped on the avatar. However you would expect that this should also remove that asset of clothing. In general these optional assets should be removable. This PR fixes this. 

![image](https://github.com/user-attachments/assets/9438ba3f-3d2c-4aa7-9d6d-abfedd4b79ac)







## How to Test

-   Run the RpmBasicLoader map
- Try euipping an outfit asset
- try removing it by selecting the same asset again
- This should not work for base model/style as that is always required
- This should also work for offline caching. So check that (You need to generate the cache first in editor before running/testing) 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
